README.md

```diff
@@ -174,6 +174,8 @@ See [awesome-malware-analysis § Books](https://github.com/rshipp/awesome-malwar
 * [HITCON](https://hitcon.org/) - Hacks In Taiwan Conference held in Taiwan.
 * [Nullcon](http://nullcon.net/website/) - Annual conference in Delhi and Goa, India.
 * [SECUINSIDE](http://secuinside.com) - Security Conference in Seoul.
+* [ARPCon 2020](https://arpcon.tech/) - three day virtual hacking conference! Exciting security talks, giveaways and also a 6 hour live jeopardy style solo CTF.
```

Comment:
Please match the style guide for this line:

* One sentence only.
* Begin with a capital letter
* Remove subjective language (e.g., "exciting").
* Do not list one-off events; reliably repeating/recurring events only, please.

```suggestion
* [ARPCon](https://arpcon.tech/) - Three day virtual hacking conference with security talks, giveaways, and a CTF.
```